pkgs/tools/bluetooth/blueman/default.nix

```diff
@@ -1,38 +1,46 @@
 { config, stdenv, lib, fetchurl, intltool, pkgconfig, python3Packages, bluez, gtk3
-, obex_data_server, xdg_utils, libnotify, dnsmasq, dhcp
-, hicolor-icon-theme, librsvg, wrapGAppsHook, gobject-introspection
+, obex_data_server, xdg_utils, dnsmasq, dhcp, libappindicator, iproute
+, gnome3, librsvg, wrapGAppsHook, gobject-introspection
+, withNetworkManager ?
+    config.networking.networkmanager.enable or false, networkmanager
 , withPulseAudio ? config.pulseaudio or stdenv.isLinux, libpulseaudio }:
 
 let
   pythonPackages = python3Packages;
-  binPath = lib.makeBinPath [ xdg_utils dnsmasq dhcp ];
+  binPath = lib.makeBinPath [ xdg_utils dnsmasq dhcp iproute ];
 
 in stdenv.mkDerivation rec {
   name = "blueman-${version}";
-  version = "2.0.8";
+  version = "2.1.1";
 
   src = fetchurl {
     url = "https://github.com/blueman-project/blueman/releases/download/${version}/${name}.tar.xz";
-    sha256 = "0kkh6jppqcn3yf70vnny1l015kxrz3dxw4g774gl02lh9ixx1bq4";
+    sha256 = "1hyvc5x97j8b4kvwzh58zzlc454d0h0hk440zbg8f5as9qrv5spi";
   };
 
   nativeBuildInputs = [
     gobject-introspection intltool pkgconfig pythonPackages.cython
-    pythonPackages.wrapPython wrapGAppsHook
-  ];
+    libappindicator pythonPackages.wrapPython wrapGAppsHook
+  ] ++ lib.optional withNetworkManager networkmanager;
```

Comment:
These should most likely go to `buildInputs`. `nativeBuildInputs` is for programs executed during build.

Reply:
Fixed!